pipeline/auto-push.cloudbuild.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: gcr.io/cloud-builders/gsutil
+    args:
+      - '-c'
```

Comment:
Should this just be

```
- scripts/upload-tests.sh
- gs://...
```

since `bash is already the entrypoint?

Reply:
SG. Will update.